stable/telegraf/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: telegraf
-version: 0.1.1
+version: 0.2.1
```

Comment:
0.1.2 is probably a better bump since there should be no functional change in a deprecation